packages/familie-typer/src/journalføring.ts

```diff
@@ -13,6 +14,21 @@ export interface IJournalpost {
     tittel?: string;
 }
 
+export interface AvsenderMottaker {
+    erLikBruker: boolean;
+    id: string;
+    land: string;
+    navn: string;
+    type: AvsenderMottakerIdType;
+}
+
+enum AvsenderMottakerIdType {
+    NULL,
```

Comment:
We tend to "string-safe" our enums,
```suggestion
    NULL = 'NULL',
```

In typescript enums are numeric by default which is a bit hard to debug :)